### DISCUSSION TITLE
XMLHttpRequest is being blocked by tauri in production build

### DISCUSSION THREAD

**Discussion by prayag17** [tauri-apps/tauri#6956] [Q&A]
I am trying to query an HTTP IP address to get some data for example `http://178.29.4.25:69` and I give `hello` as the data but when I use fetch API or axios to get the data from the URL in production it throws the error.

![image](https://github.com/tauri-apps/tauri/assets/55829513/0a3a572d-57bc-4a48-8333-fa5b3686d38d)

How do I fix this?

**Comment by FabianLars** [comment]
This is the most relevant and most active issue about this: https://github.com/tauri-apps/tauri/issues/3007 - at the bottom there are also a few workaround most (or all?) of them using the localhost plugin.
